en/concept/flight_modes.md

```diff
@@ -1,34 +1,36 @@
 # Flight Modes
 
-**Flight Modes** define the state of the system at any given time.  The user transitions between flight modes via switches on the remote control or the [ground control station](../qgc/README.md).
+**Flight Modes** define the state of the system at any given time.  The user transitions between flight modes via switches on the remote control or the [ground control station](../qgc/README.md), and are loosely grouped according to their level of control of the vehicle by the system vs. the user: "manual" modes, where the controller is essentially a pass-through from the user's remote control to the vehicle's actuators; "assisted" modes, where the system's outputs are coordinated with the user's commands; and, "auto" modes, where the controller requires little to no user input whatsoever.
```

Comment:
Hi @JonasVautherin @bkueng 
The restructure into bullet points works, but I don't find the descriptions helpful. How about:

----

* "Manual" modes are those where the user has direct control over the vehicle via the RC control (or joystick). Vehicle movement always follows stick movement, but the level/type of response changes depending on the mode.  For example, experienced fliers can use modes that provide direct passthrough of stick positions to actuators, while beginners will often choose modes that are less responsive to sudden stick-position changes.
* "Assisted" modes are also user controlled but offer some level of "automatic" assistance - for example, automatically holding position/direction, against wind. Assisted modes often make it much easier to gain or restore controlled flight.
* "Auto" modes are those where the controller requires little to no user input (e.g. to takeoff, land and fly missions).